tests/typecheck/test_decorators.yml

```diff
@@ -24,3 +24,8 @@
         from rest_framework.decorators import api_view
         @api_view()  # E: Value of type variable "_View" of function cannot be "Callable[[Any], List[Any]]"
         def view_func2(request) -> list: ...
+
+-   case: permission_classes
+    main: |
+        from rest_framework.decorators import permission_classes
```

Comment:
We need to test that `&` form works. You can also add tests for other ways if
- they are missing
- you have the time for it 🙂